deployer/handlers.go

```diff
@@ -90,19 +92,25 @@ func DeployHandler(awsc aws.AwsClients) interface{} {
 		}
 
 		release.Success = to.Boolp(true)
-		release.UnlockRoot(awsc.S3Client(nil, nil, nil))
+		locker := dynamodb.NewDynamoDBLocker(awsc.DynamoDBClient(nil, nil, nil))
+		release.UnlockRoot(awsc.S3Client(nil, nil, nil), locker, getLockTableNameFromContext(ctx, "-locks"))
 
 		return release, nil
 	}
 }
 
 func ReleaseLockFailureHandler(awsc aws.AwsClients) interface{} {
 	return func(ctx context.Context, release *Release) (*Release, error) {
-
-		if err := release.UnlockRoot(awsc.S3Client(nil, nil, nil)); err != nil {
+		locker := dynamodb.NewDynamoDBLocker(awsc.DynamoDBClient(nil, nil, nil))
+		if err := release.UnlockRoot(awsc.S3Client(nil, nil, nil), locker, getLockTableNameFromContext(ctx, "-locks")); err != nil {
 			return nil, errors.LockError{err.Error()}
 		}
 
 		return release, nil
 	}
 }
+
+func getLockTableNameFromContext(ctx context.Context, postfix string) string {
```

Comment:
I'm unsure if we want this to be available as a utility to external callers.

Reply:
Also from observation, the task name is passed in as the lambda name when testing, so the table name is actually slightly different across the different tasks.

Reply:
There is only one lambda per deployer, meaning that the task shouldn't change this. I am on the fence about making this a public method, it is small enough that other deployers can copy paste.